_data/blogs.yml

```diff
@@ -42,3 +42,8 @@
   title: Interview with Prasenjit Gautam, GCI Contestant and Open Source Contributor
   img: 7.jpg
   author_name: Kuba Michalski  
+  
+  - url: https://introductorymusicblockstutorial.wordpress.com/
+  title: Interview with Senel Demintha, GCI Contestant and Fossasia user
+  img: 7.jpg
```

Comment:
Indentation is still not correct, this is not your image, please upload an image for your blog.

Reply:
okay